### PR TITLE
S3T1 Build the page

### PR DESCRIPTION
**Pull request by mallowmew**

### CODE REVIEW

**Comment by mallowmew** [comment]
Screenshot 2019-09-17 at 11 34 19
This is how it looks with an image in it.
Thsi work may need a second pass when the API is up, when images of different sizes and aspect ratios start coming in to be displayed.

File: index.php [hunk 1]
```diff
@@ -1,2 +1,51 @@
...
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Top Cat</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div class="header">
+      <div class="container">
+        <h1>Top Cat</h1>
+        <form action="post">
```
Comment (MotNosniktaw) [inline index.php]: GET not POST

File: index.php [hunk 2]
```diff
@@ -1,2 +1,51 @@
...
+          <div class="cat-image">
+            <img src="cat-pic-placeholder.jpg" alt="" srcset="">
+          </div>
+          <div class="cat-image">
+            <img src="cat-pic-placeholder.jpg" alt="" srcset="">
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="footer">
```
Comment (MotNosniktaw) [inline index.php:37]: what is the purpose of the footer? should i t have some content in it?

File: index.php [hunk 3]
```diff
@@ -1,2 +1,42 @@
...
+          <div class="form-group">
+            <label>Select your breed:</label>
+            <div class="selector">
+              <select name="breed" id="select-breed">
+                <option value="0">Please select:</option>
+                <?php ?>
+              </select>
+            </div>
+          </div>
+        </div>
```
Comment (MotNosniktaw) [inline index.php]: div and form elements are improperly nested

**Pull request merged**